internal/discord/commands.go

```diff
@@ -41,7 +41,55 @@ func (d *Discord) Unmute(s *discordgo.Session, i *discordgo.InteractionCreate) {
 //	user:		User
 //	reason:		string
 func (d *Discord) Ban(s *discordgo.Session, i *discordgo.InteractionCreate) {
-	return
+	optionMap := mapOptions(i)
+	logMsg, _ := d.LogCommand(i.Interaction)
+
+	state := &InteractionState{
+		session: 	 s,
+		interaction: i,
```

Comment:
this is definitely a problem since it's properly formatted on my IDE, but not here